### DISCUSSION TITLE
Reconfiguring Logstash

### DISCUSSION THREAD

**Discussion by kalpinus** [telekom-security/tpotce#1088] [Q&A] [answered]
Hello,

according to [https://github.com/telekom-security/tpotce/wiki/Reconfigure-logstash.conf](url) file logstash.conf should be copy from /etc/logstash/ not from /etc/logstash/conf.d/ since directory /etc/logstash/conf.d/ is empty. If this exactly correct then should I put following line: /data/elk/logstash.conf:/etc/logstash/logstash.conf rather than /data/elk/logstash.conf:/etc/logstash/conf.d/logstash.conf in /opt/tpot/etc/tpot.yml ?

Thank you

**Comment by t3chn0m4g3** [comment, marked as answer]
Thanks. I adjusted the Wiki, since the folders have changed (`conf.d` is not used). Also an Ubuntu image is used now, which means you need to use `bash` instead of `ash`.
